mavis/DTable.h

```diff
@@ -266,6 +267,9 @@ class DTable
                 ifact->getInfoBypassCache(mnemonic, ex_info.clone());
             inst = allocator(info->opinfo, info->uinfo, std::forward<ArgTypes>(args)...);
         }
+        for(const auto& [field, value] :special_fields_map_){
+            inst->setSpecialField(field.first,field.second);
```

Comment:
What happens if you try to set a special field that doesn't exist? Does it throw an error?